src/main/features/response/views/check-and-send.njk

```diff
@@ -63,6 +63,38 @@
       {% endif %}
 
       {{ row(label = 'Why do you reject all of this claim?', value = rejectAllOption, changeLink = ResponsePaths.defenceRejectAllOfClaimPage.evaluateUri({ externalId: claim.externalId }), bold = true) }}
+
+      {% if draft.rejectAllOfClaim.option === RejectAllOfClaimOption.ALREADY_PAID %}
+        {{
+          row(
+            label = 'How much have you paid?',
+            value = t(draft.rejectAllOfClaim.howMuchHaveYouPaid.amount | numeral),
+            changeLink = FullRejectionPaths.howMuchHaveYouPaid.evaluateUri({ externalId: claim.externalId }),
+            bottomBorder = false,
+            bold = true)
+        }}
+
+        {{
+          row(
+            label = 'When did you pay this amount?',
+            value = t(draft.rejectAllOfClaim.howMuchHaveYouPaid.date.toMoment() | date),
+            bottomBorder = false,
+            bold = true
+          )
+        }}
+
+        {{
+          row(
+            label = 'How did you pay?',
+            value = t(draft.rejectAllOfClaim.howMuchHaveYouPaid.text),
+            bold = true
+          )
+        }}
+
+        {% if draft.rejectAllOfClaim.howMuchHaveYouPaid.amount < claim.totalAmountTillToday %}
+          {{ row(label = 'Why do you disagree with the amount claimed?', value = t(draft.rejectAllOfClaim.whyDoYouDisagree.text), changeLink = FullRejectionPaths.whyDoYouDisagreePage.evaluateUri({ externalId: claim.externalId }), bold = true) }}
```

Comment:
The same here, do no pass `draft.rejectAllOfClaim.whyDoYouDisagree.text` though `t()`